examples/with-npm/apps/docs/package.json

```diff
@@ -6,20 +6,23 @@
     "dev": "next dev --port 3001",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "eslint . --max-warnings 0"
```

Comment:
Is this right? Did `next lint` go away?

Reply:
In a monorepo context, I've found it easier to drop it and use ESLint with the plugin directly instead.